cli/src/declarations.ts

```diff
@@ -453,6 +453,16 @@ export interface CapacitorConfig {
      * @default true
      */
     handleApplicationNotifications?: boolean;
+
+    /**
+     * On iOS 16.4 and greater, enable debuggable web content for release builds.
+     *
+     * This is always `true` for development builds.
+     *
+     * @since 5.0.0
```

Comment:
```suggestion
     * @since 4.8.0
```

Since it's going to be released in 4.8.0 too, the since should be 4.8.0 as that's going to be the first version with the feature